modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -149,8 +219,6 @@ def getitem_array(self, key):
             )
         return self.getitem_column_array(key)
 
-    # Merge
```

Comment:
This change doesn't seem to relate to the issue.

Reply:
removed